packages/upgrade/package.json

```diff
@@ -39,7 +39,6 @@
     "chalk": "^4.1.1",
     "change-case": "^4.1.2",
     "esbuild": "^0.14.10",
-    "execa": "^5.1.1",
```

Comment:
I think this came from a garbled up merge bringing in the latest from `main`. I'll push a fix